### DISCUSSION TITLE
[New] Eviter les publicités

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#59] [Bonnes pratiques] [answered]
Les publicités sur les sites web attirent l'attention et augmentent donc le temps d'utilisation du service. A cela s'ajoute des contenus qui n'apportent rien au service.
Doit-on ajouter une bonne pratique spécifique à l'élimination des publicités (plus côté fonctionnelle) ou spécifier ce cas dans la bonne pratique "~Eliminer~ Eviter les fonctionnalités inutiles"?

**Comment by rlemaire** [comment]
On touche potentiellement au business model du site ici.
Je suis à titre personnel convaincu que le modèle de la gratuité contre de la publicité est à la source de la plupart des problèmes d'internet et devrait être remis en question mais c'est une sujet qui dépasse un peu le scope d'une checklist d'éco conception web.
J'ai proposé une pratique « limiter les tracking et analytics » (https://github.com/cnumr/best-practices/discussions/22) qui est liée au sujet de la publicité.

**Reply by devcrafting** [reply]
Un peu des 2: comme dit Thomas les pubs peuvent induire un temps d'usage plus important. Dans l'autre proposition, c'est plus l'aspect technique qui est visé : éviter le téléchargement de ressources inutiles au service (tracker, etc)
Peut-être aborder ces 2 points dans la même BP?

**Reply by ACTLEM** [reply]
@rlemaire tu as raison, cela touche plus au business model. Peut-être que le terme éliminer est trop fort car il y a publicité intrusive qui fait perdre du temps car pour certaines bloquent la navigation et publicité plus respectueuse de la navigation A voir si cette nouvelle proposition peut satisfaire: "Eviter les publicités intrusives (ou autre terme)"

**Reply by Trankilloman** [reply]
De manière générale, il faut éviter les "éliminer" pour remplacer par des "éviter", en effet, il n'est pas impossible (et ce serait normal) qu'il existe des contres exemples pour chaque BP selon l'UF du système étudié.

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: Visé par d'autres pratiques (Tracking, éliminer fonctionnalités non essentielles)
